server/webhook.go

```diff
@@ -34,7 +34,23 @@ func getStringFromTemplate(tmpl string, data any) (string, error) {
 	return result.String(), nil
 }
 
+func (p *Plugin) authenticateWebhook(r *http.Request) error {
+	token := r.Header.Get("X-MM-Cloud-Plugin-Auth-Token")
```

Comment:
I'm not married to this header name

Reply:
Yeah, 0/5 on this as well. Let's see if Juho has any input.

Reply:
This should be fine.